Makefile

```diff
@@ -13,7 +13,7 @@ clean:
 	rm -rf ./smoke-tests/report.*
 	./gradlew clean
 
-project_version:=$(shell grep 'project.version =' build.gradle | awk -F\" '{ print $$2 }')
+project_version:=$(shell ./gradlew properties -q | grep "version:" | awk '{print $$2}')
```

Comment:
One reason we had this grepping instread of gradlew'ing is that not all jobs—chiefly `publish_github`—have a Java/Gradle runner.

Reply:
hmm

Reply:
I can test it

Reply:
Doesn't seem to matter: https://app.circleci.com/pipelines/github/honeycombio/honeycomb-opentelemetry-java/981/workflows/2057538d-b7c1-4b86-aa53-c90c148b8622

Reply:
Unless we add that Make target to a job that doesn't have Java

Reply:
A-ha! Yes, true. I was misremembering that some step was using the make target to determine version but wasn't running Java.

Since we're running gradle in the target now, there's also the option of using the gradle task `project_version` that was added below the version assignment?

Reply:
that doesn't really have as nice a grep:

```sh
> Task :project_version
0.8.2-SNAPSHOT

Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

See https://docs.gradle.org/7.2/userguide/command_line_interface.html#sec:command_line_warnings

BUILD SUCCESSFUL in 726ms
1 actionable task: 1 executed
```

Reply:
I flailed trying to get the gradle task to work longer than I want to admit. I think maybe running a task requires  something extra that properties does not? I've run out of f's so I just reverted it back to properties.

Can you do another stamp please?

Reply:
Yawp. Done run out of f's.